tests/bluetooth/host/cs/mocks/zephyr/devicetree_generated.h

```diff
@@ -0,0 +1,10 @@
+/*
+ * Copyright (c) 2024 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+/*
+ * Dummy header file to avoid compiler errors
+ * intentionally left blank
+ */
```

Comment:
Which compiler error? Is it something that should be reported and fixed later?

Reply:
I copied these over from a different host unit test suite. I'm not sure if they're all really needed, let me check.

Reply:
seems not :)